app/assets/stylesheets/profile.css.scss

```diff
@@ -96,6 +96,13 @@
     &.new {
       .verify, .properties, .mod-holder { display: none; }
     }
+    .verify form { display: inline; }
+    .verify form div { display: inline; }
+    .verify input {
+      color: #428bca;
+      padding: 0;
+      font-size: 13px;
+    }
```

Comment:
**nitpick:** Since this is scss, it could be written using nesting as:
```sass
.verify {
  form { 
    display: inline; 
    div { display: inline; }
  }
  input {
    color: #428bca;
    padding: 0;
    font-size: 13px;
  }
}
```